project/.travis/before_install_test.sh.twig

```diff
@@ -29,6 +29,18 @@ fi
 
 sed --in-place "s/\"dev-master\":/\"dev-${TRAVIS_COMMIT}\":/" composer.json
 
+# symfony/maker-bundle only works with PHP 7 and higher
```

Comment:
A `TODO` to remove it when PHP5 will not be supported at all should be added here.